### PR TITLE
fix: wrong row id

### PR DESCRIPTION
**Pull request by youngsofun**
I hereby agree to the terms of the CLA available at: https://docs.databend.com/dev/policies/cla/

## Summary

reason: 

https://github.com/datafuselabs/databend/issues/14973

https://github.com/datafuselabs/databend/pull/14983

fix  a small bug about row id (shown in error msg) by the way.

## Tests

- [ ] Unit Test
- [x] Logic Test
- [ ] Benchmark Test
- [ ] No Test  - _Explain why_

## Type of change

- [x] Bug Fix (non-breaking change which fixes an issue)
- [ ] New Feature (non-breaking change which adds functionality)
- [ ] Breaking Change (fix or feature that could cause existing functionality not to work as expected)
- [ ] Documentation Update
- [ ] Refactoring
- [x] Performance Improvement
- [ ] Other (please describe):

### CODE REVIEW

**Comment by BohuTANG** [comment]
@youngsofun How about wait https://github.com/datafuselabs/databend/pull/15043? If it works as good as the tsv, disabling it won't be necessary.

**Comment by youngsofun** [comment]
yes,  I`ll test it.

> @youngsofun How about wait #15043? If it works as good as the tsv, disabling it won't be necessary.

File: src/query/settings/src/settings_default.rs [hunk 1]
```diff
@@ -207,7 +207,7 @@ impl DefaultSettings {
                     range: Some(SettingRange::Numeric(0..=u64::MAX)),
                 }),
                 ("enable_new_copy_for_text_formats", DefaultSettingValue {
-                    value: UserSettingValue::UInt64(1),
+                    value: UserSettingValue::UInt64(0),
```
Comment (youngsofun) [inline src/query/settings/src/settings_default.rs:210]:
```suggestion
                    value: UserSettingValue::UInt64(1),
```

**Comment by youngsofun** [comment]
@BohuTANG  

I reverted the change to setting, the remaining of the pr as a fix should be merged,  in case we need to used it later.

https://github.com/datafuselabs/databend/pull/15018/commits/ebb1ac122018a7fb241b918a4ec98edbb58545f7

**Review by BohuTANG — APPROVED** [review]

**Pull request merged**